File: python/ex24.py
n = int(input())
cont = 0
soma = 0
while cont < n:
    num = int(input())
    if cont == 0:
        maior = num
        menor = num
    elif num > maior:
        maior = num
    elif num < menor:
        menor = num
    soma += num
    cont += 1
print(maior)
print(menor)
print(soma)

File: c/ex24.c
//Lê n números, determinado pelo usuário, imprime o maior, o menor, a soma
#include<stdio.h>
int main(){
    int n, i = 1, soma = 0, maior, menor;
    scanf(" %i", &n);
    while(i <= n){
        int aux;
        scanf(" %i", &aux);
        if(i == 1){
            maior = aux;
            menor = aux;
        }
        else if(aux > maior){
            maior = aux;
        }
        else if(aux < menor){
            menor = aux;
        }
        soma += aux;
        i++;
    }
    printf("%i\n%i\n%i", maior, menor, soma);
    return 0;
} 